### ISSUE TITLE
Theme UI and MDX Rendering 

### ISSUE THREAD

**Issue by cjenaro** [jlengstorf/gatsby-intermediate#7]
So, another issue I found out today and here I will leave my dependencies: 

```json
  "dependencies": {
    "@emotion/core": "^10.0.27",
    "@mdx-js/mdx": "^1.5.5",
    "@mdx-js/react": "^1.5.5",
    "gatsby-plugin-mdx": "^1.0.73",
    "gatsby-plugin-theme-ui": "^0.3.0",
    "gatsby-source-filesystem": "^2.1.48",
    "mdx-utils": "^0.2.0",
    "mkdirp": "^1.0.3",
    "prism-react-renderer": "^1.0.2",
    "react-live": "^2.2.2",
    "theme-ui": "^0.3.1"
  }
```

The dependencies used by you: 

```json
"dependencies": {
    "@emotion/core": "^10.0.21",
    "@mdx-js/mdx": "^1.5.1",
    "@mdx-js/react": "^1.5.1",
    "gatsby-plugin-mdx": "^1.0.52",
    "gatsby-plugin-theme-ui": "^0.2.43",
    "gatsby-source-filesystem": "^2.1.33",
    "mdx-utils": "^0.2.0",
    "mkdirp": "^0.5.1",
    "prism-react-renderer": "^1.0.2",
    "react-live": "^2.2.1",
    "theme-ui": "^0.2.44"
  }
```

There is one of these that breaks, or maybe the combination, been trying without success to actually pinpoint what it is, but at some point, theme and components stop being overriden, so my main issue was being able to use the Code component on mdx.

I guess for now I need to downgrade my dependencies to what you are using

**Comment by jlengstorf** [comment]
this might also be related to #6 — are the versions the same across the theme(s) and site? check `yarn outdated` to make sure all the instances of the deps are at latest to avoid mismatching them

let me know if that doesn't work and please link to the repo and we can dig in further

thanks!

**Comment by cjenaro** [comment]
Thank you @jlengstorf will try this tomorrow in the morning!

**Comment by cjenaro** [comment]
This was fixed with the #6 fix, thank you
